lldb/unittests/DIL/Inputs/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+# Build `test_binary.cc` and put the binary in the Inputs folder, 
+# allowing `lldb_private::GetInputFilePath` to find it.
+get_target_property(EXE_PATH LLDBDILTests RUNTIME_OUTPUT_DIRECTORY)
+add_custom_command(
+    OUTPUT test_binary.bin
+    COMMAND $<TARGET_FILE:clang> test_binary.cc -std=c++17 -fuse-ld=lld -B$<TARGET_FILE:lld> 
```

Comment:
```suggestion
    COMMAND $<TARGET_FILE:clang> ${CMAKE_CURRENT_SOURCE_DIR}/test_binary.cc -std=c++17 -fuse-ld=lld -B$<TARGET_FILE:lld> 
```

Reply:
I found that I needed to tell clang what directory to look in, for building test_binary.cc